client/main.js

```diff
@@ -80,12 +80,27 @@ $(document).ready(function() {
 
       $('#report-issue').unbind('click');
       $('#report-issue').on('click', function() {
-          var textMessage = 'https://github.com/freecodecamp/freecodecamp/issues/new?&body=Challenge ' + window.location.href + ' has an issue.';
-          textMessage += ' User Agent is: <code>' + navigator.userAgent + '</code>.';
-          textMessage += ' Please describe how to reproduce this issue, and include links to screenshots if possible.%0A%0A';
-          textMessage = textMessage.replace(/;/g, ',');
+          var textMessage = 'Challenge [' + (challenge_Name || challengeName || window.location.href) + '](' + window.location.href + ') has an issue.\n'
+                          + 'User Agent is: <code>' + navigator.userAgent + '</code>.\n'
+                          + 'Please describe how to reproduce this issue, and include links to screenshots if possible.\n\n';
+
```

Comment:
I thought I had it enabled, but turns out that since I didn't have `eslint-plugin-react` and `babel-eslint` installed globally, they just failed without any errors..
It looks like there are  a lot of errors in the file; if you want to, you could merge this and I'll clean up everything (big promises) and submit another PR as soon as I can?